distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestFederation.java

```diff
@@ -56,7 +58,27 @@
  */
 @RunWith(PaxExam.class)
 @ExamReactorStrategy(PerClass.class)
-public class TestNsili extends AbstractIntegrationTest {
+public class TestFederation extends AbstractIntegrationTest {
```

Comment:
As a counterpoint, the DDF project has 321 files named `*Test.java` compared to 185 `Test*.java`.

My preference would be to push toward the `*Test.java` naming convention for two reasons:

1. When doing a file search/using IDEs to open classes, a search for `MyClass` will find both `MyClass.java` and `MyClassTest.java` quickly and easily.
2. I know that the majority of the DDF and Alliance developers use Intellij for software development; when using the `CTRL-Shift-T` (`CMD-Shift-T` on Mac) to create a new test, the naming pattern it uses is `*Test`. If we opt for `Test*`, using that shortcut will require the additional step of editing the filename prior or after to creation.